crates/re_viewer/src/ui/override_ui.rs

```diff
@@ -0,0 +1,266 @@
+                        temporary_style_ui_for_component(ui, component_name, |ui| {
+                            item_ui::component_path_button(
+                                ctx,
+                                ui,
+                                &ComponentPath::new(entity_path.clone(), *component_name),
+                            );
+                        });
+                    });
+                    // Editor last to take up remainder of space
+                    row.col(|ui| {
+                        let component_data = match store_kind {
+                            StoreKind::Blueprint => {
+                                let store = ctx.store_context.blueprint.store();
+                                let query = ctx.blueprint_query;
+                                get_component_with_instances(
+                                    store,
+                                    query,
+                                    entity_path,
+                                    *component_name,
+                                )
+                            }
+                            StoreKind::Recording => get_component_with_instances(
+                                store,
+                                &query,
+                                entity_path,
+                                *component_name,
+                            ),
+                        };
+
+                        if let Some((_, _, component_data)) = component_data {
+                            ctx.component_ui_registry.edit(
+                                ctx,
+                                ui,
+                                UiVerbosity::Small,
+                                &query,
+                                store,
+                                entity_path,
+                                &overrides.override_path,
+                                &component_data,
+                                instance_key,
+                            );
+                        } else {
+                            // TODO(jleibs): This shouldn't happen. Warn instead?
+                            ui.weak("(empty)");
+                        }
+                    });
+                }
+            });
+        });
+}
+
+pub fn add_new_override(
+    ctx: &ViewerContext<'_>,
+    query: &LatestAtQuery,
+    store: &DataStore,
+    ui: &mut egui::Ui,
+    space_view: &SpaceViewBlueprint,
+    data_result: &DataResult,
+    active_components: &BTreeSet<ComponentName>,
+) {
+    ui.menu_button("Add new override", |ui| {
+        ui.style_mut().wrap = Some(false);
+
+        let view_systems = ctx
+            .space_view_class_registry
+            .new_visualizer_collection(*space_view.class_identifier());
+
+        // We have to have at least 1 visualizer system or we can't create an override
+        let Some(system_for_initial_value) = view_systems.systems.values().next() else {
+            return;
+        };
```

Comment:
so if we have several systems that could provide an initial valuewe just ignore those?

Reply:
For now, yes. We could do something more complex in the future but holding off on this until we understand the context.